cypress/features/regression/accessibility/accessibilityDesignSystem.feature

```diff
@@ -104,28 +99,28 @@ Feature: Accessibility tests - Design System folder
     When I open "Design System <component>" component page "default story" in no iframe
     Then "<component>" component has no accessibility violations
     Examples:
-      | component         |
-      | Badge             |
-      | Batch Selection   |
-      | Checkbox          |
-      | Content           |
-      | Date Input        |
-      | Date Range        |
-      | Draggable         |
-      | Fieldset          |
-      | Flat Table        |
-      | Hr                |
-      | Loader            |
-      | Pager             |
-      | Search            |
-      | Select            |
-      | Select Filterable |
-      | Switch            |
-      | Tabs              |
-      | Textarea          |
-      | Textbox           |
-      | Text Editor       |
-      | Tile Select       |
+      | component             |
+      | Advanced Color Picker |
+      | Badge                 |
+      | Batch Selection       |
+      | Content               |
+      | Draggable             |
```

Comment:
This is missing `Date Input` and `Date Range`

Reply:
Those tests are / should be in `accessibilityCommon.feature` file